.all-contributorsrc

```diff
@@ -0,0 +1,92 @@
+{
+  "projectName": "logdna-winston",
+  "projectOwner": "logdna",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "angular",
+  "contributors": [
```

Comment:
This whole block can come out if we're using `all-contributors`, no? I'm not aware of `all-contributors` auto-updating this block.